lib/src/Pos/Launcher/Resource.hs

```diff
@@ -229,10 +229,10 @@ getRealLoggerConfig LoggingParams{..} = do
     pure $ overrideConsoleLog $ cfg <> cfgBuilder
   where
     overrideConsoleLog :: LoggerConfig -> LoggerConfig
-    overrideConsoleLog = case lpConsoleLog of
-        Nothing    -> identity
-        Just True  -> (<>) (consoleActionB defaultHandleAction)
-        Just False -> (<>) (consoleActionB (\_ _ -> pass))
+    overrideConsoleLog src = case lpConsoleLog of
+        Nothing    -> src
+        Just True  -> src <> consoleActionB defaultHandleAction
+        Just False -> src <> consoleActionB (\_ _ -> pass)
```

Comment:
Sooooo. Lost a good two hours on that when I was trying to prevent nodes from logging into the console. We introduced this function to allow an overriding of the logging behavior. Node that this behavior can already be tweaked from the logger config files directly (`printOutput: False`) but this gets overwritten by some hard-coded config in the lines above. Yet, our override function wasn't working.properly; logger configs are semi-groups and the `lcConsoleAction` is a `Last` functor, so the order in which we append them matters. Hence the fix.